delfin/drivers/hpe/hpe_3par/component_handler.py

```diff
@@ -53,8 +50,7 @@ def get_storage(self, context):
                 # Check the hardware and software health
                 # status of the storage system
                 # return: System is healthy
-                command_str = ComponentHandler.HPE3PAR_COMMAND_CHECKHEALTH
-                re_str = self.sshclient.doexec(context, command_str)
+                re_str = self.sshhanlder.get_health_state()
```

Comment:
Is the health state not available through rest?

Reply:
Rest has no interface to check health status, only SSH command line support

Reply:
By looking at the official documents, it is found that rest does not support the interface of checking health status and obtaining all alarm information. At present, it can only be obtained through SSH command line

Reply:
Ok